example/src/App.tsx

```diff
@@ -270,6 +297,46 @@ const App = () => {
             }
           })()
         }
+        <AppButton
+          title={isNativeUIMRecShowing ? 'Hide Native UI MRec' : 'Show Native UI MRec'}
```

Comment:
For non-code, we usually uppercase the whole thing (that's how we do it in the docs as well). Just a convention we made up yearsss ago:

```suggestion
          title={isNativeUIMRecShowing ? 'Hide Native UI MREC' : 'Show Native UI MREC'}
```